plugin/lspconfig.lua

```diff
@@ -0,0 +1,91 @@
+if vim.g.lspconfig ~= nil then
+  return
+end
+vim.g.lspconfig = 1
+
+local version_info = vim.version()
+if vim.fn.has 'nvim-0.7' ~= 1 then
```

Comment:
has(nvim-0.7) will be true for all versions 7 or greater. It will only be false for 0.6 or older. The check should be has('nvim-0.8')

Reply:
I am a bit confused. The minimum required version is 0.7. it should fail if its lower than 0.7. but if this is set to 0.8, wont it fail for all versions other than nightly?

Reply:
my mistake, I thought you wanted to check for 0.8